scss/_settings_animations.scss

```diff
@@ -13,8 +13,11 @@ $animation-easing: (
 );
 
 @mixin vf-transition($property: all, $duration: brisk, $easing: out) {
+  @if (type-of($property) != string) {
+    @error "vf-transition $property must be a string";
+  }
   transition-duration: map-get($animation-duration, $duration);
-  transition-property: $property;
+  transition-property: unquote($property);
```

Comment:
If you pass quoted strings or list to `vf-transition` as `$property` it'll result in an invalid CSS (and scss does not complain).
 
As an example, passing either `$property: ('width', 'box-shadow')` or `'width, box-shadow'` will have a result similar to the following:
<img width="489" alt="image" src="https://user-images.githubusercontent.com/7452681/186116644-c3ce8696-8516-47b7-bd8a-466048cd6e2d.png">

@bartaz In cases such as this, would you prefer if we formatted the string (e.g. by using unquote) or simply threw an error and disallow passing incorrectly formatted strings in the first place?

Reply:
Formatting the string seems better.
I guess in most cases anyone trying to use it would search for examples of use and copy those. But I agree it is a little bit unclear how to pass multiple prop to animate at once.